pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -1016,3 +1014,18 @@ def _is_slurm_managing_tasks(self) -> bool:
         total_requested_devices = (self.num_gpus or self.num_processes) * self.num_nodes
         num_slurm_tasks = int(os.environ["SLURM_NTASKS"], 0)
         return num_slurm_tasks == total_requested_devices
+
+    def _check_tpu_mis_config(self) -> None:
+        # TODO moved from TPUAccelerator when refactor accelerator. Revisit when refactor
+        # accelerator_connector @four4fish
```

Comment:
@four4fish I would avoid such TODOs (from experience with other refactors) and rather note them to the related issue.

Reply:
Yeah make sense, will remove